jetpack.php

```diff
@@ -65,6 +67,7 @@ function jetpack_should_use_minified_assets() {
 add_filter( 'jetpack_should_use_minified_assets', 'jetpack_should_use_minified_assets', 9 );
 
 // @todo: Abstract out the admin functions, and only include them if is_admin()
+require_once( JETPACK__PLUGIN_DIR . 'jetbench.php'                    );
```

Comment:
Should we move that file into a folder, to avoid adding one more file to the already busy Jetpack root? 

Maybe we should place it in the theme tools directory, and add it to the `$tools` array in `module-extras.php` when `JETPACK__BENCHMARK_COOKIE` is defined?

In any case, you may have to rename the file so IDEs don't scream. :)